haystack/finder.py

```diff
@@ -51,6 +52,10 @@ def get_answers(self, question: str, top_k_reader: int = 1, top_k_retriever: int
             results = {"question": question, "answers": []}
             return results
 
+        import collections
+        if isinstance(documents[0], collections.Iterable):
```

Comment:
All the retrievers should return a list of `Document` instances. The `TfidfRetriever` was breaking this behavior and will now be fixed soon by #73. So, we would not need this conversion.

Reply:
If it is merged it should not affect the fix in #73 and can be removed later